package.json

```diff
@@ -13,7 +13,8 @@
   "homepage": "https://thelounge.github.io/",
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "npm run build:font-awesome && npm run build:grunt && npm run build:handlebars",
+    "build:font-awesome": "mkdir -p client/fonts && cp node_modules/font-awesome/fonts/fontawesome-webfont.* client/fonts",
```

Comment:
Yeah... Instead of copying files and running scripts, couldn't we make it so the server builds/serves all that stuff on demand? That way it would also leave room for automatic rebuilds and ship more stuff from npm.

Or maybe make one script that deals with all the building without using the shell so much...